src/org_parser/transform.cljc

```diff
@@ -45,17 +46,31 @@
                    first
                    count)
         title (->> properties
-                   (property :title)
-                   (str/join " "))]
+                   (property :text))
+        tags (->>  []) ;; TODO extract tags from the last element of :text if it's a :text-normal; empty otherwise;
```

Comment:
This line actually does not only extract tags, it also transforms the last `:text-normal` list element. Therefore, write a helper function which returns both, title and tags.